vivisect/__init__.py

```diff
@@ -49,6 +50,10 @@
 
 import vivisect.analysis.generic.emucode as v_emucode
 
+def guid():
```

Comment:
perhaps you could use the `uuid` module that's part of the standard library?

https://docs.python.org/2/library/uuid.html